sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_day_6_v1/query.sql

```diff
@@ -51,7 +45,16 @@ client_conditions AS (
     attribution_source,
     distribution_id,
     attribution_ua,
-    COALESCE(udf.bitcount_lowest_7(days_seen_bits), 0) >= 5 AS activated
+    -- did the profile send a main ping in at least 5 of their first 7 days?
+    COALESCE(BIT_COUNT(mozfun.bits28.from_string('1111111000000000000000000000') & days_seen_bits) >= 5, FALSE) AS activated,
```

Comment:
there are other ways to do this but for me its always been the most readable way. fine to change it